### DISCUSSION TITLE
离线版增加自定义披风功能

### DISCUSSION THREAD

**Discussion by XuanNap** [Meloong-Git/PCL#347] [提问] [answered]
龙猫龙猫，隔壁HMCL自定义披风都不知道出现多少月了，您打算什么时候加入PCL啊

**Comment by WForst-Breeze** [comment]
好久没用HMCL了，它出自定义披风这个事我还真没听说过（）你指的是离线登录的自定义披风吗？刚刚去HMCL试了一下也没有找到自定义披风的通道。如果是我眼瞎没找到请麻烦指引一下（）但是根据我现有的一点点知识来说，我还真想不到有什么办法（第三方登录除外）能支持离线模式的自定义披风。
但是如果你指的是微软登录（即正版登录）的披风更换的话，请右键头像，单击“更换披风”以更换你的披风

**Reply by WForst-Breeze** [reply]
补充：如果您指的是第三方登录的话，请前往 版本设置-设置-服务器选项-登录方式-第三方登录：Authlib Injector 或 LittleSkin -填写下方的“认证服务器（必填）” 即可

**Comment by zkitefly** [comment]
自定义皮肤他指的应该是这个
![image](https://user-images.githubusercontent.com/64117916/204064621-bcaad37c-4e8d-401a-b9dd-f43493c9036c.png)

**Comment by WForst-Breeze** [comment, marked as answer]
我的锅我的锅，眼瞎没看见HMCL的自定义披风……已经有人反馈了我才想起来，请见 #700
